services/galley/src/Galley/API/LegalHold.hs

```diff
@@ -403,4 +407,6 @@ changeLegalholdStatus tid uid lhStatus = do
     chunksOf maxSize uids = case splitAt maxSize uids of (h, t) -> h : chunksOf maxSize t
 
     blockConnection :: UserId -> UserId -> Galley ()
-    blockConnection = error "TODO"
+    blockConnection userA userB = do
```

Comment:
what's the difference between blocking a connection and blocking the corresp. 1:1 conversation?